### ISSUE TITLE
Batch norm in WGAN discriminator

### ISSUE THREAD

**Issue by KhrystynaFaryna** [cyclomon/3dbraingen#6]
Hi, I was wondering why did you guys decide to use batch norm in a critic of WGAN-GP. The paper on Improved training of WGAN(the one where gradient penalty is proposed) advises against it. 
Thanks in advance!

**Comment by cyclomon** [comment]
Hi, 
Although we noticed that it is not a good choice of using BN in discriminator of WGAN-GP,
we found that using batchnorm stabilized training. 
We also experimented with Instance Norm , LayerNorm, and without normalization, but we could not find a better model. 
Now we moved to another project so we still don't know the exact reason why this works. 
Maybe further experiments and analysis will be needed to figure it out.
Thanks!
